build.sh

```diff
@@ -172,6 +172,7 @@ export WPA_PASSWORD
 export WPA_COUNTRY
 export ENABLE_SSH="${ENABLE_SSH:-0}"
 export PUBKEY_ONLY_SSH="${PUBKEY_ONLY_SSH:-0}"
+export SSH_PORT="${SSH_PORT:-22}"
```

Comment:
Since the default port is already 22, why fallback to setting this variable to 22? If you do this, the `-n` condition will not be true and you'll modify the default sshd_config even if there's no change necessary. Then users will be bugged about conffile changes any time sshd_config changes in the debian package.

Reply:
I'll change to `export SSH_PORT`